docs/development.md

```diff
@@ -1,15 +1,61 @@
-Development of Unfolded plugin
-===========================
+Plugin development
+==================
 
-This project uses [qgis_plugin_tools](https://github.com/GispoCoding/qgis_plugin_tools) submodule, so when cloning
-use `--recurse-submodules` like so:
-`git clone --recurse-submodules https://github.com/UnfoldedInc/qgis-plugin.git`
+## Setup
```

Comment:
Maybe add a note that these instructions are tested on mac, and also indicate last updated data, what MacOS version and QGIS version you used.

Reply:
Done in https://github.com/foursquare/qgis-plugin/pull/62/commits/9ed8907c135726ff63fafff203d295effc8ece33